bom-2.204.x/pom.xml

```diff
@@ -17,6 +17,16 @@
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
+            <dependency>
+                <groupId>io.jenkins</groupId>
+                <artifactId>configuration-as-code</artifactId>
+                <version>1.36</version>
+            </dependency>
+            <dependency>
+                <groupId>io.jenkins.configuration-as-code</groupId>
+                <artifactId>test-harness</artifactId>
+                <version>1.36</version>
+            </dependency>
```

Comment:
Could this not be replaced by?
```xml
        <configuration-as-code-plugin.version>1.36</configuration-as-code-plugin.version>
```

Reply:
I’m not sure, don’t really know how properties work in inherited Poms like this

Reply:
It’s not something that should ever change as this is the last version that supports that line

Reply:
see #315